<launch>
  <node name="guarded_node" pkg="exercise1" type="guarded_node" output="screen" />
  <param name="stop_dist" type="double" value=".75" />
  <param name="tolerance" type="double" value=".0001" />
  <include file="$(find turtlebot_stage)/launch/turtlebot_in_stage.launch" />
  <node pkg="turtlebot_teleop" type="turtlebot_teleop_key" name="turtlebot_teleop_keyboard" output="screen" >
    <param name="scale_linear" value="0.5" type="double" />
    <param name="scale_angular" value="1.5" type="double" />
    <remap from="turtlebot_teleop_keyboard/cmd_vel" to="/user_cmd_vel" />
  </node>
</launch>
